### ISSUE TITLE
`--code-style` argument will cause error

### ISSUE THREAD

**Issue by darwin67** [ScaCap/action-ktlint#48]
## Summary

`1.2.0` got released today, and CI is failing for us using `latest`.

```
ktlint version: ktlint version 1.2.0
Parameter '--code-style' is no longer valid. The code style should be defined as '.editorconfig' property 'ktlint_code_style='
reviewdog: parse error: EOF
```

Seems like this has been deprecated for a while now, so perhaps time to remove it?

## Expected behavior

CI runs

## Current behavior

`ktlint` will return an error without executing.

**Comment by cp-megh-l** [comment]
+1

**Comment by ffuf-libre** [comment]
+1

**Comment by guil-camp-scacap** [comment]
Until it is fixed, a workaround to get builds working again is to explicitly set the ktlint version to a previous one (i.e. `1.1.1`)

[Sample commit](https://github.com/ScaCap/service.franz.payment/pull/220/commits/8b5d97a0f5ae2f9c725148b569612b2456775ede#diff-b8a0058f42de9f56e68376aea04ada87bf6cd7e3a9a7f4eadc5671bad78d40f8R18)

**Comment by ghaiszaher** [comment]
Thanks for reporting. The action should not be using `--code-style` for versions after 1.0.0. However, in the new ktlint version, the command `ktlint --version` doesn't simply return the version `1.2.0` but rather `ktlint version 1.2.0` for some reason. So the comparison is now failing.

Our goal in the entrypoint script was to make it work with any ktlint version so we have a logic of comparing versions and passing different values for android support.

Should be fixed with https://github.com/ScaCap/action-ktlint/pull/50

**Comment by ghaiszaher** [comment]
Fixed with #51
